admin_manual/configuration/files/external_storage/auth_mechanisms.rst

```diff
@@ -39,6 +39,21 @@ does not work.
    (See :doc:`../../server/background_jobs_configuration`.) Be
    aware that the Ajax cron mode is triggered by browsing the ownCloud Web GUI.
 
+Known Limitations
+~~~~~~~~~~~~~~~~~
+
+Please be aware that because of the way that credentials are stored in ownCloud, any operations must be performed by the logged-in mount owner, because credentials are not stored anywhere.
```

Comment:
what about: ``are stored in ownCloud`` --> ``are stored in ownCloud community version``

Reply:
Q: is this at least true for personal shared mounts. Is it also true for admin shared mounts or is it the same?

Reply:
this sentence sounds contradictory because first it talks about "the way that credentials are stored" then later it says "not stored anywhere"

the limitation exists for both admin and personal mounts where both have the "log-in credentials, save in session" option